components/ScheduleForm.js

```diff
@@ -0,0 +1,112 @@
+import styled from "styled-components";
+import { useState } from "react";
+import { useRouter } from "next/router";
+
+const StyledButton = styled.button`
+  display: block;
+  width: 50%;
+  margin: 10px auto;
+  padding: 10px 24px;
+  border: 3px solid var(--secondary-stroke-color);
+  background-color: var(--secondary-bg-color);
+  border-radius: 1rem;
+  color: white;
+  font-weight: bold;
+  font-size: 1rem;
+  cursor: pointer;
+  &:hover {
+    color: var(--secondary-stroke-color);
+    background-color: white;
+  }
+`;
+const FormContainer = styled.form`
+  display: grid;
+  gap: 0.5rem;
+`;
+const Label = styled.label`
+  font-weight: bold;
+`;
+const Select = styled.select`
+  width: 100%;
+  border: 3px solid black;
+  border-radius: 0.5rem;
+  padding: 0.5rem;
+  margin: 0.5rem 0;
+`;
+const Input = styled.input`
+  width: 100%;
+  padding: 0.5rem;
+  border: 3px solid black;
+  border-radius: 0.5rem;
+  margin: 0.5rem 0;
+`;
+
+export default function ScheduleForm({
+  plants,
+  handleAddReminder,
+  isMyPlantFunction,
```

Comment:
thats a strange name for a prop, maybe you want to remove the "function" from the name?